extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -101,6 +101,13 @@ private function getUserById( $userId ) {
 			throw new \InvalidArgumentException( self::NO_USER_MATCH_ERROR );
 		}
 
+		$user->load();
```

Comment:
`newFromId()` does not load any user information into the class besides the user id.